.gitignore

```diff
@@ -11,3 +11,4 @@ sample/
 *.swp
 *.swo
 *.vim
+__azurite*.json
```

Comment:
Would we only expect this for Storage? Should this be in a .gitignore under sdk/storage in that case just to keep the root .gitignore a little smaller / more manageable? (.gitignore files "stack").